### PR TITLE
Feature/add loading spinner and message if no data

### PR DESCRIPTION
**Pull request by jhommark**

### CODE REVIEW

File: client/app/dashboard/model/list/ModelList.js [hunk 1]
```diff
@@ -468,39 +478,39 @@ angular.module('dashboard.Dashboard.Model.List', [
...
-                      delete newRow[relationship.model];
-                    }
-                  }
-                }
-              }
-            }
-          }
-          */
-          
+           if ($scope.model && $scope.model.options.relations) {
```
Comment (tma-isbx) [inline client/app/dashboard/model/list/ModelList.js]: Whitespace clean up broke the formatting of this comment.

**Comment by tma-isbx** [comment]
Please resolve conflicts, thanks!

**Comment by tma-isbx** [comment]
@jhommark Looks good 👍

@coolwr Should be ready for your review and merge after @jhommark resolves conflicts. Much of the diff is due to whitespace clean up. The actual relevant changes are fairly short.

**Comment by jhommark** [comment]
Hi @tma-isbx and @coolwr,

I updated the branch to remove the whitespaces cleanup that happened and adjusted by my IDE. I patched it with my new local branch to put only the relevant changes.

**Comment by tma-isbx** [comment]
@coolwr Ready for your review and merge.

**Pull request merged**